front/src/views/routes/common/cms/CommonCMSPage.vue

```diff
@@ -31,12 +31,13 @@
           :id="container.id"
           :key="container.id"
           class="lck-container"
-          :class="{
-            'lck-elevation': container.elevation
-          }"
         >
-          <div v-show="page.modeNavigation === 'tab' ? container.id === currentHash : true">
-            <h2 v-if="container.display_title" class="lck-color-title">
+          <div
+            v-show="page.modeNavigation === 'tab' ? container.id === currentHash : true"
+            class="p-px-4"
+            :class="{'lck-elevation': container.elevation}"
+          >
+            <h2 v-show="container.display_title" class="lck-color-title">
```

Comment:
Why do you use a `v-show` and not a `v-if` ?

Reply:
Because in print, we display container titles regardless of whether they are displayed on the page or not.